internal/cli/auth/login.go

```diff
@@ -173,6 +173,7 @@ func (opts *LoginOpts) setUpProfile(ctx context.Context) error {
 	// Initialize the text to be displayed if users are asked to select orgs or projects
 	opts.OnMultipleOrgsOrProjects = func() {
 		if !opts.AskedOrgsOrProjects {
+			_, _ = fmt.Fprintln(opts.OutWriter, "Now set your default organization and project.")
```

Comment:
[q] Are we still showing this message when the user has only one org and one proj?

Reply:
No, we should not show it as they'll not select anything